### PR TITLE
Docker free tests

### PR DESCRIPTION
**Pull request by slawosz**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -40,20 +39,25 @@ jobs:
...
-        - secure: "nA9iN+1zxXm1PpcxEBumfgDCJgg9Mrpa0d4kK0a9BxLDrucG0pakPoM7EOP8xzBULdxSfdU8fF1CoSaODqQLl8eB60VW0A+S6S8fk5/cAKszXQuv3yud8n5EL2+QqCexDLuDxDumNWPNpS0mFexSRMpFS90Qx3XXUBHuOFQ2V3AXZaXBJnhCrJqvP7/2Qd9yVHhKA5DBirzDa7LZYHYrWuTxHuqYLdi3f9r4ExpVLeSk3V5UYB73xntEi6p556bb8H6qpgs5P+JX3UntyPCUWVZrIDOjTA3t5AdhPDoQunrHh8ophHjzyiSeoQOxd8MIV1SnX1Gk6hEQrlSuHW+4AzG4xrk8H42Vz7WnpGUEuUc28AZpGZHkQnsAqv07hinGz9nmZPcSWXP6FdLJNKQhBAuo1da6O8+cTv/Hh+a7YCO9q9EhAQi6TIFgqbz8Z3QoWgwIfqlJOHdPcvePV1a8mQ1tYNDQ/V+72iHglNMtsAhG7DsTz2ueTnwvwG+H9bRK2rSNfzSS/2uV/AQLKnM9H3DQaymDFFuxhkitZiI9c7g9nC/8ZOWsK5zaMNODS9CtcRQOw3sWCOre5Id2gy7XWtQcVTVrkw0PeLCtJNBExPSCGD2pd8NCuKxErtjZUhiOpb9hMWyix8IFxpbYRfLWCfz0aeMpWl7lfe4eXuSXCwU=" # CF_PASSWORD
+        - DATABASE_URL=postgres://localhost/psd_test?user=postgres
```
Comment (StupidCodeFactory) [inline .travis.yml]:
```suggestion
        - DATABASE_URL=postgres://postrges@localhost:5432/psd_test
```

File: .travis.yml [hunk 2]
```diff
@@ -1,9 +1,8 @@
 sudo: required
 
 language: generic
-
 services:
-  - docker
+  - postgresql
```
Comment (StupidCodeFactory) [inline .travis.yml]: - redis
      - elasticsearch

File: .travis.yml [hunk 3]
```diff
@@ -40,20 +39,25 @@ jobs:
 
     - stage: Test and deploy review
       name: PSD Tests
-      script: ./infrastructure/ci/test.sh
+      script:
+        - cd psd-web
+        - bundle install
```
Comment (StupidCodeFactory) [inline .travis.yml]:
```yaml
- bundle check | bundle install --deployment --jobs 4 
```
 Then you want to save/restore `vendor/bundle` afer/before starting the build

**Comment by slorek** [comment]
Now redundant due to removal of Travis from CI workflow.

**Pull request closed without merging**